### PR TITLE
express-post-json exercise - Ariel Donner

### PR DESCRIPTION
**Pull request by arieldonner**
express-post-json

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+
+const grades = [];
```
Comment (scbowler) [inline express-post-json/index.js:6]: `grades` should be an object

File: express-post-json/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  res.json(gradesArr);
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  grades.push(req.body);
+  grades[grades.length - 1].id = nextId;
+  nextId++;
+  res.sendStatus(201);
```
Comment (scbowler) [inline express-post-json/index.js:22]: With `grades` as an object you have to create a new property in the object using the `nextId` as the key. For example:
```JAVASCRIPT
app.post('/api/grades', (req, res) => {
  const newGrade = req.body;
  const id = nextId;
  newGrade.id = id;
  grades[id] = newGrade;
  nextId++;
  res.status(201).json(newGrade);
});
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Heading in the right direction, but `grades` should be an object. Let me know if you have any questions.

File: express-post-json/index.js (hunk 2 above)
Comment (arieldonner) [reply express-post-json/index.js:22]: Thank you! That helps a lot. It makes sense that it should be like the previous grades object.

**Pull request merged**